platform/security/saml/saml-assertion-validator-impl/src/main/java/com/connexta/ddf/security/saml/assertion/validator/impl/SamlAssertionValidatorImpl.java

```diff
@@ -116,78 +111,59 @@ protected DocumentBuilder initialValue() {
   String signatureProperties = null;
 
   /**
-   * Validates a SAMLAuthenticationToken by checking it's signature against the configured system
+   * Validates a SAML SecurityAssertion by checking it's signature against the configured system
    * certs.
    *
-   * @param token token to validate
+   * @param securityToken saml BaseAuthenticationToken containing saml assertion to validate
    * @throws AuthenticationFailureException thrown when the cert fails to validate
    */
   @Override
-  public void validate(SAMLAuthenticationToken token) throws AuthenticationFailureException {
+  public void validate(SecurityToken securityToken, X509Certificate[] certs, String requestUri)
```

Comment:
📘 Refactored to allow the passing in of a Saml Assertion as a `SecurityToken` rather than as a `SamlAuthenticationToken`.

Reply:
I don't think we should expose a CXF class in our interface.